reference/features/filtering_and_faceted_search.md

```diff
@@ -263,6 +263,10 @@ Like any other filter, attributes you want to use as facets must be added to the
 
 Once they have been configured, you can search for facets with the `filter` search parameter.
 
+:::warning
+Please note that **synonyms don't apply to facets.** Meaning, if you have `SF` and `San Francisco` set as synonyms, filtering by `SF` and `San Francisco` will show you **different results.**
```

Comment:
Since facets are just a subset of filters and that facets are harder to explain, what do you think about this:

```suggestion
Please note that **synonyms don't apply to filters.** Meaning, if you have `SF` and `San Francisco` set as synonyms, filtering by `SF` and `San Francisco` will show you **different results.**
```